owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/operations/RemoteOperation.java

```diff
@@ -121,7 +121,7 @@ public Thread execute(OwnCloudClient client,
         if (listener == null) {
             throw new IllegalArgumentException
                     ("Trying to execute a remote operation asynchronously " +
-                            "without a listener to notiy the result");
+                            "without a listener to notify the result");
```

Comment:
This can be written within the same line, don't you think?

Btw, there are more lines in this file that can be written just in one line.

Reply:
Of course, and not only in this file, but this is out of scope of this pull request. Keep the focus to the commit message